libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -38,11 +38,12 @@ const AP_Param::GroupInfo AC_Avoid::var_info[] = {
 };
 
 /// Constructor
-AC_Avoid::AC_Avoid(const AP_AHRS& ahrs, const AP_InertialNav& inav, const AC_Fence& fence, const AP_Proximity& proximity)
+AC_Avoid::AC_Avoid(const AP_AHRS& ahrs, const AP_InertialNav& inav, const AC_Fence& fence, const AP_Proximity& proximity, const AP_Beacon* beacon)
     : _ahrs(ahrs),
```

Comment:
This is for other vehicle which is using AC_Avoidace. It doesn't affect to other vehicle by setting default value nullptr.